Assets/UI/Scripts/OKButton.gd

```diff
@@ -0,0 +1,2 @@
+extends ButtonBase
+class_name OKButton
```

Comment:
Currently this doesn't do anything. Typically I would recommend against committing empty scripts, and waiting until you have content for the scripts to commit them, but it's fine. Anyway, this review comment is a reminder to add content to these later.

Also, the GDScript style guide suggests placing `class_name` above `extends`.